akd_quorum/src/node/mod.rs

```diff
@@ -0,0 +1,2217 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
Well.. I think, some code walk-through Zoom call would be useful.

Reply:
Agreed, like I mentioned earlier this was to serve as a jumping off point and be ultimately broken into separate PRs of smaller scale. It proved painful to not try and design all the traits/messages/crypto together in a single branch, but now that we have a relatively decent idea, we should be able to break it apart reasonably.